packages/okta-vue/README.md

```diff
@@ -231,6 +231,23 @@ router.beforeEach((to, from, next) {
 - `redirect_uri` **(required)**: Where the callback is hosted
 - `scope` *(optional)*: Reserved or custom claims to be returned in the tokens
 - `response_type` *(optional)*: Desired token grant types
+- [`storage`](#storage) *(optional)*: Specify the type of storage for tokens
+- [`auto_renew`](#auto_renew) *(optional)*: Defaults to true, enables tokens auto renewal
+
+#### storage
+
+Specify the type of storage for tokens.
+The types are:
+- [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+- [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+- [`cookie`](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie)
+
+Defaults to `localStorage` and will fall back to `sessionStorage`, and/or `cookie` if no one of the previous type is available.
```

Comment:
Is there a reason we don't default to sessionStorage instead? OWASP seems to recommend it: https://www.owasp.org/index.php/JSON_Web_Token_(JWT)_Cheat_Sheet_for_Java#Token_storage_on_client_side

Reply:
sessionStorage doesn't span tabs, so I've always found it a terrible user experience.